app/main.c

```diff
@@ -279,7 +279,13 @@ int app_main(bool is_resale)
 	}
 #endif
 #if defined SELF_SIGNED_CERTS_SUPPORTED
-	if  (argc > 1 && (*argv[1] == '5' || *argv[2] == '5')) {
+	int strcmp_ss = 1;
+	int strcmp_ss2 = 1;
+
+	strcmp_s((char *)argv[1], DATA_CONTENT_SIZE, "'-ss'", &strcmp_ss);
+	strcmp_s((char *)argv[2], DATA_CONTENT_SIZE, "'-ss'", &strcmp_ss2);
+
+	if  (argc > 1 && (strcmp_ss || strcmp_ss2)) {
```

Comment:
[out] | indicator | pointer to result indicator, greater than 0, equal to 0 or less than 0, if the string pointed to by dest is greater than, equal to or less than the string pointed to by src respectively.

Looks like the expected value of the 4th argument strcmp_ss is zero ?